### PR TITLE
Feature/jenkins2

### PR DESCRIPTION
**Pull request by polyakrecords**
[VSTS](https://msmobilecenter.visualstudio.com/Mobile-Center/Krukow-M1-Team/_workitems/edit/40033)

- Updated project for blue ocean CI

### CODE REVIEW

**Comment by polyakrecords** [comment]
Jenkins test this please

**Comment by polyakrecords** [comment]
@jmoody we need to remove old jenkins CI check from this repo

File: calabash-cucumber/test/cucumber/run.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 bundle update
 
-bundle exec cucumber -t ~@flick -f pretty -f json -o reports/instruments.json
+bundle exec cucumber -t ~@flick -f pretty -f json -o reports/instruments.json -f junit -o reports/junit
```
Comment (jmoody) [inline calabash-cucumber/test/cucumber/run.sh]: Remove the `-f json -o reports/instruments.json`

**Comment by polyakrecords** [comment]
Jenkins test this please

**Pull request merged**